__tests__/app.test.js

```diff
@@ -83,4 +80,73 @@ describe("API testing", () => {
         });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    test("return correct article when passed an article id ", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+          expect(Object.keys(body.articles)).toHaveLength(8);
+        });
+    });
+    test("for articles with invalid ids we should expect a 404 error", () => {
```

Comment:
this is not non-existent, rather than invalid; invalid suggests something bad about the data type / request